staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -538,7 +533,7 @@ func NewKubectlCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 	i18n.LoadTranslations("kubectl", nil)
 
 	// From this point and forward we get warnings on flags that contain "_" separators
-	cmds.SetGlobalNormalizationFunc(cliflag.WarnWordSepNormalizeFunc)
+	cmds.SetGlobalNormalizationFunc(cliflag.WordSepNormalizeFunc)
```

Comment:
I'm not sure we want to silently drop attempts to use deprecated flags  (the separated with _).

I suggest keeping here as 
`cmds.SetGlobalNormalizationFunc(cliflag.WarnWordSepNormalizeFunc)`